test/Project.toml

```diff
@@ -2,6 +2,8 @@
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
 OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
```

Comment:
```suggestion
```
We don't need OrdinaryDiffEq.jl anymore, do we?

Reply:
We do if we want this version to be compatible with v0.10 of Trixi.jl (e.g., for other open PRs to Trixi.jl).

Reply:
Ah, I see.